Python/PalsonKansakar/task2/sqlite.py

```diff
@@ -0,0 +1,31 @@
+import csv
+import sqlite3
+
+def create_table(cursor):
+    cursor.execute("""
+        CREATE TABLE weather(period text,descriptions text,temperature text)
+    """)
+
+def insert_data(path,cursor):
+    insert = "INSERT INTO weather (period, descriptions, temperature)  \
+    VALUES (?, ?, ?)"
+
+    with open(path, 'r') as file:
+        data = csv.DictReader(file)
+        next(data) 
+        for row in data:
+            cursor.execute(insert, (row['period'],row['descriptions'],row['temperature']))
+
+
+def main():
+    path = './outputs/weather.csv'
+    db = '/home/palson/db/scrapersqlite.db'
+    con = sqlite3.connect(db)
+    cursor = con.cursor()
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager